plotly_kaleido/Cargo.toml

```
@@ -22,9 +22,9 @@ dunce = "1.0.2"
 directories = "4.0.1"
 
 [dev-dependencies]
-zip = "0.5.13"
+zip = "1.1.1"
```

```suggestion
zip = "1"
```
Overly-precise version specifications cause Dependabot to open a PR for each patch release, rather than for each major release.